app/components/TabBarIcon/TabBarIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React, {FC} from 'react';
```

Comment:
Added proper rules for checking a type, so you do not miss adding import type as a type.
Also, do proper eslint & prettier config because currently not proper code formatting